lblod/ember-rdfa-editor
# Internal/standarization and classes rework

### Overview
First part of the standarization ticket, add a class to every node that can be changed by the user and document how.

##### connected issues and PRs:
GN-4644


### Setup
None

### How to test/reproduce
Mostly a review of API, but you can add classes to the different nodes and check that they get applied, also you can add styles to the existing classes

### Challenges/uncertainties
None



### Checks PR readiness
